WebRoot/js/ajax/dwt/widgets/DwtListView.js

```diff
@@ -862,6 +863,12 @@ function(item, skipNotify, forceSelection) {
 	}
 };
 
+DwtListView.prototype._updateAriaLabel =
```

Comment:
Note: This function get called when user select/deselect item in the list (by clicking checkbox or with shortcut key CTRL+` ). This function will add/remove `selected` text from `aria-label` as per user selection such that NVDA tool will announce it properly.